### PR TITLE
Remove old MC-Inscale reports

### PR DESCRIPTION
**Pull request by lwyszomi**
@czue @snopoke

### CODE REVIEW

File: custom/reports/mc/models.py [hunk 1]
```diff
@@ -1,633 +0,0 @@
-from datetime import timedelta
```
Comment (snopoke) [inline custom/reports/mc/models.py]: Does django not require there to be a models file?

**Comment by lwyszomi** [comment]
@snopoke 
Added models file.

**Comment by snopoke** [comment]
build failure real
```
======================================================================
FAIL: test_instantiate_all (testapps.test_pillowtop.tests.test_settings.PillowtopSettingsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/testapps/test_pillowtop/tests/test_settings.py", line 35, in test_instantiate_all
    self.assertEqual(len(all_pillow_configs), len(expected_meta))
AssertionError: 48 != 49
```

Also, have you run this locally and verified that the new reports still work?

**Comment by lwyszomi** [comment]
@snopoke 
Sorry I forgot that we have tests for fluff's. I fixed test but still fails, I think now is not related to my changes.

I checked new reports and all works correctly.

**Comment by snopoke** [comment]
build should get fixed by https://github.com/dimagi/commcare-hq/pull/12208

**Comment by lwyszomi** [comment]
@snopoke 
Okay, so i'm waiting for merge your changes, then I will update my PR.

**Comment by czue** [comment]
looks good to me, though did you mean to check in `custom/eqa/__init__.py`?

**Comment by lwyszomi** [comment]
@czue @snopoke 
I added this by mistake :) Also I updated PR with the newest master branch

**Pull request merged**